apps/api/src/message-schemas/commands/register-representation.schema.json

```diff
@@ -20,6 +20,10 @@
 			"type": "string",
 			"description": "The unique reference of the case"
 		},
+		"representationType": {
```

Comment:
Needs to be added to `required` array the top if we want it to be required. If not then we need to make it optional in the interface type file.